src/metrics-modeling/tests/test_pipeline.py

```
@@ -0,0 +1,119 @@
+        encoding="utf-8",
+    )
+    # SPORTDEETS_SECRETS_PATH may point at the PowerShell secrets FILE,
+    # not its directory — the loader handles both.
+    monkeypatch.setenv("SPORTDEETS_SECRETS_PATH", str(tmp_path / "_common-variables.ps1"))
+
+    from metricbot import config as config_module
+
+    values = config_module._load_env_file()
+    assert values["METRICBOT_PG_HOST"] == "db.example.com"
+    assert values["METRICBOT_PG_USER"] == "bob"
+    assert values["METRICBOT_PG_PASSWORD"] == "p@ss=word"
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to: "METRICBOT_PG_PASSWORD"

(S105)